test/AlgebraicGeometry/Schemes/elliptic_surface.jl

```diff
@@ -146,3 +146,85 @@ end
+  # But it broke the tests at some point leading to timeout, 
+  # so we put it here to indicate regression.
+  for (i, g) in enumerate(values(gluings(default_covering(X))))
+    gluing_domains(g) # Trigger the computation once
+  end
+
+  D_P = Oscar._section(X, P)
+ 
+  II = first(components(D_P));
+ 
+  trans = Oscar.translation_morphism(X, P; divisor=D_P)
+ 
+  JJ = Oscar._pushforward_section(trans, P; divisor=D_P)
+ 
+  IIX = first(components(Oscar._section(X, 2*P)));
+  @test IIX == JJ
+  
+  # pushforward of the whole algebraic lattice with verification of the result.
+  # This test is hardcoded, but should remain stable throughout future changes.
+  # Details can be found in a forthcoming paper on Oguiso's automorphism 
+  # by Simon Brandhorst and Matthias Zach
+  lat = algebraic_lattice(X)[1]
+  A = [intersect(a, b) for a in lat, b in lat]
+ 
+  ll = Oscar._pushforward_lattice_along_isomorphism(trans)
+  res_mat = [Oscar.basis_representation(X, d) for d in ll]
+  ref_mat = [
+             [ 1,  0,   0,   0,   0,   0,   0,   0,   0,   0,  0,   0,  0,   0,   0,   0,   0,   0],
+             [ 7,  3,  -1,  -1,  -2,  -1,  -1,  -1,  -1,  -2,  1,   0,  0,  -1,  -1,  -1,   0,   0],
+             [ 1,  0,  -1,  -1,  -1,   0,   0,   0,   0,   0,  0,   0,  0,   0,   0,   0,   0,   0],
+             [ 0,  0,   0,   0,   1,   0,   0,   0,   0,   0,  0,   0,  0,   0,   0,   0,   0,   0],
+             [ 0,  0,   0,   1,   0,   0,   0,   0,   0,   0,  0,   0,  0,   0,   0,   0,   0,   0],
+             [ 0,  0,   0,   0,   0,   0,   1,   0,   0,   0,  0,   0,  0,   0,   0,   0,   0,   0],
+             [ 0,  0,   0,   0,   0,   1,   0,   0,   0,   0,  0,   0,  0,   0,   0,   0,   0,   0],
+             [ 0,  0,   0,   0,   0,   0,   0,   0,   1,   0,  0,   0,  0,   0,   0,   0,   0,   0],
+             [ 0,  0,   0,   0,   0,   0,   0,   1,   0,   0,  0,   0,  0,   0,   0,   0,   0,   0],
+             [ 1,  0,   0,   0,   0,   0,   0,  -1,  -1,  -1,  0,   0,  0,   0,   0,   0,   0,   0],
+             [ 9,  4,  -1,  -2,  -3,  -2,  -2,  -2,  -2,  -2,  0,   0,  0,  -1,  -1,  -1,   0,   0],
+             [ 4,  2,   0,   0,  -1,  -1,  -1,   0,   0,  -1,  1,   0,  0,   0,  -1,  -1,   0,   0],
+             [ 6,  2,  -1,  -1,  -1,  -1,  -1,   0,  -1,  -2,  1,  -1,  1,   0,  -1,  -1,   0,   0],
+             [ 6,  2,   0,  -1,  -2,  -1,  -1,  -1,  -1,  -1,  1,   1,  0,  -1,  -1,  -1,   0,   0],
+             [11,  4,  -1,  -2,  -3,  -2,  -2,  -1,  -1,  -2,  1,   0,  0,  -1,  -1,  -2,   0,   0],
+             [11,  4,  -1,  -2,  -3,  -2,  -2,  -1,  -1,  -2,  1,   0,  0,  -1,  -2,  -1,   0,   0],
+             [11,  4,  -1,  -1,  -2,  -2,  -2,  -2,  -2,  -2,  1,   0,  0,  -1,  -1,  -1,  -1,   0],
+             [10,  4,  -1,  -2,  -3,  -2,  -2,  -1,  -1,  -2,  1,   0,  0,  -1,  -1,  -1,   0,  -1]
+            ]
+  @test_broken res_mat == ref_mat # TODO: Simon, are you sure about the form of ref_mat?
```

Comment:
@simonbrandhorst : The good news: All computations go through now rather quickly. The bad news: Comparing with the matrix you sent me on Slack the other day, we still get different results. Can you have a look whether I interpreted everything correctly?

Reply:
I checked that the resulting matrix `res_mat` is indeed orthogonal and gives the correct intersection matrix. One could also add this as a (working) test here. But the translation by this particular section does not seem to produce the base change you anticipated.